examples/local-transactions/local_transactions.description

```diff
@@ -1,5 +1,6 @@
 // A Ballerina transaction is a series of data manipulation statements 
 // that must either fully complete or fully fail, thereby leaving the
-// system in a consistent state. This sample uses H2 database. Please
-// change the DB connection properties as required.
+// system in a consistent state. This sample uses H2 database. Prior
+// to running this sample, create these databases and change the DB 
```

Comment:
No need to create the databases before running the sample. They will create during the execution of the sample.